doc/geany.txt

```diff
@@ -4580,14 +4580,14 @@ line_wrap_indent
 
 translucency
     Translucency for the current line (first argument) and the selection
-    (second argument). Values between 0 and 256 are accepted.
+    (second argument). Values between 0 and 255 are accepted.
 
     Note for Windows 95, 98 and ME users:
-    keep this value at 256 to disable translucency otherwise Geany might crash.
+    keep this value at 255 to disable translucency otherwise Geany might crash.
 
     Only the first and second arguments are interpreted.
 
-    *Example:* ``translucency=256;256;false;false``
+    *Example:* ``translucency=255;255;false;false``
```

Comment:
Maybe it could say "Only two arguments are interpreted" and then not show the `;false;false` in the example?